packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -5,13 +5,23 @@
 import {HttpServer} from '../../';
 import {supertest, expect} from '@loopback/testlab';
 import * as makeRequest from 'request-promise-native';
-import {ServerRequest, ServerResponse} from 'http';
+import {ServerRequest, ServerResponse, get, IncomingMessage} from 'http';
 
 describe('HttpServer (integration)', () => {
   let server: HttpServer | undefined;
 
   afterEach(stopServer);
 
+  it('formats IPv6 url correctly', async () => {
+    server = new HttpServer(dummyRequestHandler, {host: '::1'});
+    await server.start();
+    if (server.address) {
```

Comment:
How can it happen that `server.address` is not set after the server started?

If really dislike tests that silently skip the actual work and pretend to complete successfully.

I am proposing to either remove this check, or change it into an assertion that fails the test if `server.address` is not set.

Unless I am missing something?